src/rtc/h5_prep.py

```diff
@@ -300,20 +309,196 @@ def get_metadata_dict(product_id: str,
+            ['source_processing_sw_version',
+             str(burst_in.ipf_version),
+             'IPF version of the source data'],
+        
+        'RTC/metadata/sourceDataInformation/azimuthLooks':  # 1.6.6
+            ['source_azimuth_looks',
+             burst_in.burst_misc_metadata.azimuth_looks,
+             'Azimuth number of looks'],
+        'RTC/metadata/sourceDataInformation/slantRangeLooks':  # 1.6.6
+            ['source_slant_range_looks',
+             burst_in.burst_misc_metadata.slant_range_looks,
+             'Slant range number of looks'],
+
+        'RTC/metadata/sourceDataInformation/productLevel':  # 1.6.6
+            ['source_product_level',
+             'L1',
+             'Product level of the source data'],
+        'RTC/metadata/sourceDataInformation/geometry':  # 1.6.7
+            ['source_geometry',
+             'slant range',
+             'Geometry of the source data'],
+        'RTC/metadata/sourceDataInformation/azimuthSpacing':  # 1.6.7
+            ['source_azimuth_spacing',
+             burst_in.azimuth_time_interval,
+             'Azimuth spacing of the source data in seconds'], 
+        'RTC/metadata/sourceDataInformation/slantRangeSpacing':  # 1.6.7
+            ['source_slant_range_spacing',
+             burst_in.range_pixel_spacing,
+             'Slant range spacing of the source data in meters'], 
+
+        'RTC/metadata/sourceDataInformation/azimuthResolution':  # 1.6.7
+            ['source_azimuth_resolution',
+             burst_in.azimuth_time_interval,
+             'Azimuth time resolution of the source data in seconds'],
+        'RTC/metadata/sourceDataInformation/slantRangeResolution':  # 1.6.7
+            ['source_slant_range_resolution',
+             burst_in.range_pixel_spacing,
+             'Slant range resolution of the source data in meters'],
+
+        'RTC/metadata/sourceDataInformation/nearRangeIncidenceAngle':  # 1.6.7
+            ['near_range_incidence_angle',
+             burst_in.burst_misc_metadata.inc_angle_near_range,
+             'Near range incidence angle in meters'],
+        'RTC/metadata/sourceDataInformation/farRangeIncidenceAngle':  # 1.6.7
+            ['far_range_incidence_angle',
+             burst_in.burst_misc_metadata.inc_angle_far_range,
+             'Far range incidence angle in meters'],
+        # Source for the max. NESZ:
+        # (https://sentinels.copernicus.eu/web/sentinel/user-guides/
+        #  sentinel-1-sar/acquisition-modes/interferometric-wide-swath)
+        'RTC/metadata/sourceDataInformation/maxNoiseEquivalentSigmaZero':  # 1.6.9
+            ['max_noise_equivalent_sigma_zero',
+             -22,
+             'Maximum Noise equivalent sigma0 in dB'], #TODO see if it can be extracted from burstNoise
+
+        'RTC/metadata/processingInformation/dataAccess':  # placeholder for 1.7.1
+            ['product_data_access',
+             'TBD',
+             'URL to access the product data'],
```

Comment:
```suggestion
        # TODO Review: should we expose this parameter in the runconfig?
        #'RTC/metadata/processingInformation/dataAccess':  # placeholder for 1.7.1
        #    ['product_data_access',
        #     'TBD',
        #     'URL to access the product data'],
```
Let's comment this for now